src/utils.cairo

```diff
@@ -2,41 +2,67 @@ use core::option::OptionTrait;
 use core::traits::TryInto;
 const PRECISION: u64 = 1_000_000;
 
+
+fn get_max_value(bit_depth: u16) -> u64 {
```

Comment:
An array could have worked too.
I assume we don't need a get_min_value, can you confirm ?

Reply:
There is no need for get_min_value as far as we aren't concerned with the amplitude.